shell/src/main/java/alluxio/cli/UnderFileSystemContractTest.java

```diff
@@ -111,42 +106,47 @@ private void runS3Operations() throws Exception {
     mUfs = UnderFileSystem.Factory.create(mUfsPath, getUfsConf());
 
     String testDir = createTestDirectory();
-    loadAndRunTests(new S3ASpecificOperations(testDir, mUfs, mConf), testDir);
+    return loadAndRunTests(new S3ASpecificOperations(testDir, mUfs, mConf), testDir);
   }
 
   /**
    * Loads and runs the tests in the target operations class.
    *
    * @param operations the class that contains the tests to run
    * @param testDir the test directory to run tests against
+   * @return the number of failed tests
    */
-  private void loadAndRunTests(Object operations, String testDir) throws Exception {
+  private int loadAndRunTests(Object operations, String testDir) throws Exception {
+    int failedTestCnt = 0;
     try {
       Class classToRun = operations.getClass();
-      Field[] fields = classToRun.getDeclaredFields();
-      for (Field field : fields) {
-        field.setAccessible(true);
-      }
```

Comment:
why remove this?

Reply:
Reverted. I made a mistake thinking these are all Methods. Tks for the catch.